tests/cephfs/cephfs_utilsV1.py

```diff
@@ -3097,3 +3098,30 @@ def nfs_mount_and_io(self, nfs_client, nfs_server, export_name, mount_path):
             f"{mount_path}/{dir_name}",
             long_running=True,
         )
+
+    def create_directory(self, client, directory):
```

Comment:
can we directly run mkdir in line 3125

Reply:
wanted to reduce the creation of directories in the test scripts by adding a common function. 
anything wrong with having this as a common function?

Reply:
There is no harm in creating function. However the same could have been used in line 118. Also, other team members remembering a function exists for creating a directory is low as it is easier to fire up a single mkdir command when needed. Reusable function should have multiple lines and good chunk of function handled is the thought process.

Reply:
Updated